index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>surveyForm</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+
+  <body>
+    <div id="title-div">
+        <h1 id="title">freeCodeCamp Survey Form</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+
+    </div>
+    <div class="container">
+    <form action="" class="survey-form" id="survey-form" method="post">
+        <fieldset>
+            <label for="name" id="name-label">Name <input type="text" name="name" id="name" placeholder="Enter your name" required></label>
+            <label for="email" id="email-label">Email <input type="email" email="email" id="email" placeholder="Enter your email" required></label>
+            <label for="age"id="number-label">Age <small>(optional)</small><input type="number" name="age"  id="number" placeholder="Age" min="16" max="100"></label>
```

Comment:
l suggest moving the small tag inside the closing label tag for better semantics and accessibility